ts/src/test/tests.ts

```diff
@@ -87,6 +87,16 @@ class testMainClass {
     }
 
     async init (exchangeId, symbolArgv, methodArgv) {
+        try {
+            await this.initInner (exchangeId, symbolArgv, methodArgv);
+        } catch (e) {
+            dump ('[TEST_FAILURE]'); // tell run-tests.js this is failure
+            throw e;
+        }
+    }
+    
```

Comment:
there were specific cases, eg. happening errors during build, however it didn't break tests. the reason was that run-tests didn't see the `TEST_FAILURE` phrase. now, it's inevitable that on any action there would be `TEST_FAILURE` in output.

so, the story is that, in cases of test failures, we are using `dump('[TEST_FAILURE]...` however there are cases when error/exception happened, but it didn't use 'dump' approach, instead plain exception were thrown, causing run-tests being unable to find `TEST_FAILURE` phrase. so, in this approach, we would always have `TEST_FAILURE` when exception is thrown (as opposed when error is `dump`-ed)